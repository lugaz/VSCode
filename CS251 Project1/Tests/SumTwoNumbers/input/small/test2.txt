9614324006671879694792407891
9494715398184721795504873
