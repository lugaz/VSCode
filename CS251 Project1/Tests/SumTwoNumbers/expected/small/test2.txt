9623818722070064416587912764
